### ISSUE TITLE
[split] version no longer functional? 

### ISSUE THREAD

**Issue by adae22** [An00nymushun/DiscordFreeEmojis#73]
Greetings!

First off, I really appreciate what you're doing here with this plugin!

I just wanted to briefly ask if [split] version is no longer working (or if it's just me)?

If so, is there any chance of an updated in the future? 
Alternatively, are there any other means to get the image link to be sent separately when it follows text (so as to avoid the link being visible)?

Again, thanks for all the hard work!

**Comment by An00nymushun** [comment]
Yes, I think I didn't update it yet. I've been thinking about rewriting all my plugins when I have the time.

Not sure how you would split your message automatically but you can do it manually.

**Comment by adae22** [comment]
Thanks for the heads up!

I kinda had a hunch you might not want to maintain X different versions of the same plugin, understandable!

Considering that I'm not knowledgeable as far as this stuff goes, and judging by your reply I'm guessing discord changed some stuff around with the inner workings? 
I remember Qb's plugin being able to split a message into two (separating the message part from the image part to prevent the image link from becoming visible) back when that plugin was still a thing, and judging by previous comments in here it seems that your [split] version used to do the same thing...

In any case thanks again for your work on all of this. 
If there was a way to (re)implement this feature in the future I'm sure a lot of people would be pretty stoked about it. 
If not, oh well, we'll just have to keep consciously separating messages from emotes ourselves.

Have a good one!
